config/experiment.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+//Experiment defines if experimental features are available
+type Experiment struct {
+	AdCerts ExperimentAdsCert `mapstructure:"adscert"`
+}
+
+//ExperimentAdsCert configures and enables functionality to generate and send Ads Cert Auth header to bidders
+type ExperimentAdsCert struct {
+	Enabled   bool             `mapstructure:"enabled"`
+	InProcess AdsCertInProcess `mapstructure:"inprocess"`
+	Remote    AdsCertRemote    `mapstructure:"remote"`
+}
+
+//AdsCertInProcess configures data to sign requests using ads certs library in core PBS logic
+type AdsCertInProcess struct {
+	//Origin is ads.cert hostname for the originating party
+	Origin string `mapstructure:"origin"`
+	//PrivateKey is a base-64 encoded private key.
+	PrivateKey string `mapstructure:"key"`
+	//DNSCheckIntervalInSeconds default: 30
```

Comment:
I like the comments explaining each of the config options. Can you please also add brief comments about what these two DNS interval configurations mean? I think it will be useful for people trying to configure their own Prebid Server as we don't have any other specific documentation explaining these config options